File: AMS_电子对账/code/source/YssSztTool/Service/Para/IErOrgService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using FAST.Common.Service.Services.Base;
using YssSztTool.Pojo.Para;
using FAST.Common.Service.Attributes;

namespace YssSztTool.Service.Para
{
    [ServiceAttribute(ServiceId = "osgi-elecreco", ServiceController = "com.yss.uco.elecreco.er.org.controller.IErOrgServiceController")]
    public interface IErOrgService : IServiceBus
    {
        [MethodAttribute]
        List<ErOrg> getTrusteeOrgs();

        [MethodAttribute] 
        List<ErOrg> getManagerOrgs();
    }
}

File: AMS_电子对账/code/server/source/com.yss.uco.elecreco/src/main/java/com/yss/uco/elecreco/er/org/service/IErOrgService.java
package com.yss.uco.elecreco.er.org.service;

import java.util.List;

import com.yss.framework.api.mvc.biz.IServiceBus;
import com.yss.framework.api.restful.annotations.RestfulSupported;
import com.yss.uco.elecreco.er.org.pojo.ErOrg;

/**
 * BUG230634电子对账界面卡顿严重
 * @author Lenovo
 *
 */
@RestfulSupported
public interface IErOrgService extends IServiceBus {
	/**
	 * 前台下拉框调用
	 * 获取托管机构
	 * @return
	 */
	public List<ErOrg> getTrusteeOrgs();
	/**
	 * 前台下拉框调用
	 * 获取管理机构
	 * @return
	 */
	public List<ErOrg> getManagerOrgs();
}
